lib/utilities/local_music_controller.dart

```diff
@@ -38,18 +40,18 @@ class localMusicController implements MusicInterface {
 
   /// lets the user choose the folder who´s files get put into the Playlist
   /// path to this folder is stored in [directoryPath]
-  Future<String> getPlaylistDir() async {
+  Future<void> setNewPlaylistDir() async {
     String? temp = await FilePicker.platform.getDirectoryPath();
     if (temp == "/" || temp == null) {
       directoryPath = "/";
     } else {
       directoryPath = temp.toString();
+      Settings().setMusicPath(directoryPath);
     }
-    return directoryPath;
   }
 
   /// Returns a List which includes all paths to the files in the directory [path]
-  Future<List<String>> loadMusicFromPath(String path) async {
+  Future<List<String>> _loadMusicFromPath(String path) async {
     Directory directory = Directory(directoryPath);
     return await directory.list().listen(
```

Comment:
RuntimeException
![image](https://user-images.githubusercontent.com/29731782/223404572-eb981ee3-dd31-4002-93d0-b5d18d35fe17.png)

Reply:
Fehler der aufgetreten ist wenn man versucht hat Dateien in einem Ordern aufzulisten